Release 1.12 plan.md

```diff
@@ -0,0 +1,33 @@
+# Plan for Version 1.12
+
+## Timing
+
+The last feature release of chakracore occurred on 29th June 2018 (version 1.10) since then there have been numerous security update releases and 1 non-security bug fix release (1.11). There has however been significant feature work performed since that date all of which is unreleased.
+
+We would therefore like our first release to be done fairly quickly - however there are also some half complete pieces of work to finish before that as well as a variety of bugs introduced that should be fixed before that.
+
+We will aim to do this first release by June to allow time for confirming the exact status of the project, completing unfinished work, performing testing and fixing bugs.
+
+## Actions prior to release
+
+We will endeavour to perform the below work prior to issuing this first community release in June. If time permits additional work may be done - conversely if time is too short the release may be produced with some of these actions not taken - in either event what has and has not been done will be explained.
+
+1. Review major changes between 1.11 and master to generate an accurate set of release notes.
+
+1. Review current master against standards to identify major gaps including assessing against the tc39 list of stage 3 and stage 4 proposals and also testing with test262. This action may identify further actions to be taken before release including fixing small deviations from the standards.
+
+1. Review all open bug reports and assess impact, fix where practicable.
+
+1. Complete partially complete work including:
+
+    - Generator, async function and async-generator function enhancements and enabling of async-iteration and jitting of all of these types of functions.
+
+    - Implementation of Top level await
+
+    - RegExp ES6 properties and unicode mode
```

Comment:
This one (RegEx) may be a larger piece of work BUT as far as I know it's the one major ES6 deficiency that Chakracore still has and IMO it just needs to be sorted.

Reply:
This is just a suggestion - to reduce scope of the changes before release, I'd consider regex "extra credit" - if we make it it would be great, but if that turns out to be lots of work, then it can go into the next release. I am fine either way - I don't understand full technical scope, but it looks like we need to review compliance.

Reply:
There's 3 parts RegEx Symbols, RegEx prototype properties AND RegEx unicode mode matching.

The first two have implementations behind features flags - I think they're 90% done, need testing, a little fixing and enabling. The third one - unicode mode matching is not done at all - the unicode flag is used for how the RegEx is parsed but the matcher seems to ignore.

Reply:
I’ll do some exploratory work on this later today to asses it further.